doc/src/thermo_style.rst

```diff
@@ -348,16 +348,16 @@ dimensions *lx*, *ly*, *lz*, *yz*, *xz*, *xy*\ .
 
 ----------
 
-For output values from a compute or fix, the bracketed index I used to
-index a vector, as in *c_ID[I]* or *f_ID[I]*, can be specified
-using a wildcard asterisk with the index to effectively specify
-multiple values.  This takes the form "\*" or "\*n" or "n\*" or "m\*n".
-If N = the size of the vector (for *mode* = scalar) or the number of
-columns in the array (for *mode* = vector), then an asterisk with no
-numeric values means all indices from 1 to N.  A leading asterisk
-means all indices from 1 to n (inclusive).  A trailing asterisk means
-all indices from n to N (inclusive).  A middle asterisk means all
-indices from m to n (inclusive).
+For output values from a compute or fix or variable, the bracketed
+index I used to index a vector, as in *c_ID[I]* or *f_ID[I]* or
+*v_name[I]*, can be specified using a wildcard asterisk with the index
+to effectively specify multiple values.  This takes the form "\*" or
+"\*n" or "n\*" or "m\*n".  If N = the size of the vector (for *mode* =
```

Comment:
This seems wrong, should be vector, and the term mode is not defined

Reply:
Thanks for catching that - I just pushed a change that removes the mode disussion.  That was apparently text copied
from fix ave/time, where this a mode option in the command.